support-frontend/assets/helpers/abTests/abtest.js

```diff
@@ -334,6 +329,23 @@ function getParticipations(
   return participations;
 }
 
+function getAmountsTestParticipations(countryGroupId: CountryGroupId, settings: Settings): ?Participations {
+  if (!settings.amounts) {
+    return null;
+  }
+  const { test } = settings.amounts[countryGroupId];
+
+  if (!test || !test.isLive) {
+    return null;
+  }
+
+  const variants = ['CONTROL', ...test.variants.map(variant => variant.name)];
```

Comment:
I'll add a check to support-admin-console to prevent a user naming a variant 'CONTROL'